src/Agent/NewRelic/Agent/Core/Transformers/TransactionTransformer/TransactionTransformer.cs

```diff
@@ -291,6 +291,13 @@ private void GenerateAndCollectMetrics(ImmutableTransaction immutableTransaction
                 MetricBuilder.TryBuildClientApplicationMetric(referrerCrossProcessId, catResponseTime, catResponseTime, txStats);
             }
 
+            // Capture required Kafka metrics
```

Comment:
Does this need to be done here specifically? It feels a bit out of place.

Reply:
I had trouble finding a good spot for this since we needed the full name.  Since all that was finalized here, it seemed like a good spot.